src/cmd/root.go

```diff
@@ -359,6 +364,35 @@ func newSubIDFileError() error {
 	return errors.New(errMsg)
 }
 
+func setUpConfiguration() error {
+	logrus.Debug("Setting up configuration")
+
+	configFiles := []string{
+		"/etc/containers/toolbox.conf",
+	}
+
+	userConfigDir, err := os.UserConfigDir()
+	if err == nil {
+		configFiles = append(configFiles, fmt.Sprintf("%s/containers/toolbox.conf", userConfigDir))
+	}
+
+	viper.SetConfigType("toml")
+
+	for _, configFile := range configFiles {
+		viper.SetConfigFile(configFile)
+
+		if err := viper.MergeInConfig(); err != nil {
+			if _, ok := err.(viper.ConfigFileNotFoundError); ok {
+				logrus.Debugf("Setting up configuration: file %s not found", configFile)
+			} else {
+				logrus.Debugf("Setting up configuration: failed to merge file %s: %s", configFile, err)
```

Comment:
We should probably error out for other errors. eg., if the configuration file can't be parsed.

Reply:
Yes, I agree.